src/prettierd.ts

```diff
@@ -5,6 +5,7 @@ import { promisify } from "util";
 const { version } = require("../package.json");
 
 const readFile = promisify(fs.readFile);
+const validCommands = ["restart", "start", "status", "stop"];
```

Comment:
This could be renamed if another naming convention is preferred. An alternative would obviously be `VALID_COMMANDS`.

Reply:
This is fine. Ideally this would be something that we could import from `core_d`, but it doesn't look like this is something they export 😞